app/account/connections/page.tsx

```diff
@@ -19,9 +20,9 @@ const ConnectionsPage: React.FC = () => {
   const orcidContentString = sanitizeHtml('ORCID provides a persistent identifier - an ORCID iD - that distinguishes you from other users. Learn more at <a href="https://orcid.org/" target="_blank" rel="noopener noreferrer">ORCID.org</a>.');
 
   return (
-    <>
-      <h1 className={styles.title}>Connections</h1>
-      <div className="container">
+    <PageWrapper title={'Connections'}>
```

Comment:
Added shared PageWrapper to add title, and shared ContentContainer for the white box background and styling